content/articles/2018-05-ggplot2-2-3-0.Rmd

```diff
@@ -0,0 +1,126 @@
+account of updates and changes.
+
+In addition to highlighting a few features and improvements, we also want to 
+share a bit about our release-preparation process for ggplot2, which has over 
+1,700 reverse dependencies.
+
+## Breaking changes
+
+### Our process
+
+Regression testing, visual and otherwise, allows us to keep track of changes
+throughout the development process. Before submitting an update to CRAN,
+you need to ensure that the packages that depend on yours still pass `R CMD
+check`, and inform reverse-dependency maintainers in the event that they do not.[^1] 
+We use [revdepcheck](https://github.com/r-lib/revdepcheck) to help automate this 
+process, and distinguish between "false positives" (pre-existing failures), and 
+issues introduced in the development version of your package. 
+
+At this point, we begin to manually examine the failures. "Newly broken" issues
+are extracted from the problems report and put into a Google sheet where (with
+the great help of Thomas Lin Pedersen, Claus Wilke, and Kara Woo) we identified
+clusters of errors to tackle. For example, 97 packages returned a warning due
+to a namespace clash between `dplyr::vars()` and `ggplot2::vars()`. That’s
+something we clearly need to fix, so we open an [issue](https://github.com/
+tidyverse/ggplot2/issues/2585) to track progress.
+
+Where reverse-dependency errors result from a deliberate design change, we now
+have a collection of "symptomatic" error messages, and (hopefully) fixes to
+offer the maintainers of affected packages. These go in the "Breaking changes"
+section of the NEWS file, so that if you encounter a new error you can easily
+search for the error message and find the resolution. Because ggplot2 is an old
+and much used package, we try to keep breaking changes to a minimum. However,
+sometimes we deem it important to make a change that break existing code.
+Typically this is for one of two reasons:
+
+ * We discover that code that previously appeared to work was actually hiding a 
+ potential error. For example, in this version of ggplot2 we ensure that input 
+ data frames don’t have duplicate column names. Previously ggplot2 just took the 
+ first column with a name, which might have concealed a bug.  
+
+ * We decided that it’s worth breaking a small amount of existing code in the 
+ interests of improving future code. For example, this version of ggplot2 adds 
+ support for [tidy evaluation](https://github.com/tidyverse/ggplot2/blob/master/NEWS.md#tidy-evaluation) 
+ making ggplot2 more compatible with the rest of the tidyverse. This breaks a 
+ small number of packages that use ggplot2 but should affect very little data 
+ analysis code, so we decided that it was worth it.  
+
+### What to do if you run into a breaking change
+
+The maintainers of reverse dependencies that are likely to encounter a breaking 
```

Comment:
I think you should remove the bit about revdeps here - this is orthogonal to what a user should do if they see a problem.